--- a/data/maya-calendar.desktop
+++ b/data/maya-calendar.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-Name=Maya
+Name=Calendar
 GenericName=Calendar
 Comment=View and schedule events
 Exec=maya-calendar
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -71,7 +71,7 @@
             build_version = Build.VERSION;
             build_version_info = Build.VERSION_INFO;
 
-            program_name = "Maya";
+            program_name = _("Calendar");
             exec_name = "maya-calendar";
 
             app_years = "2011-2013";
